Sources/KituraSampleRouter/RouterCreator.swift

```diff
@@ -20,7 +20,8 @@ import Foundation
 
 import Kitura
 import KituraMustache
-import KituraStencil
+import KituraStencil // required for using StencilTemplateEngine
```

Comment:
I tried this and `git diff` showed no difference. Perhaps it is  GitHub anomaly.